.github/workflows/e2e.yml

```diff
@@ -1,42 +1,73 @@
 name: e2e Tests
+
 on: [push]
+
 jobs:
   cypress-run-ubuntu:
     runs-on: ubuntu-20.04
-    # There is currently a bug on chrome 100 that adds an extra space in the editor
-    # container: cypress/browsers:node14.17.6-chrome100-ff98
-    # container: cypress/browsers:node14.17.0-chrome88-ff89
-    container: cypress/browsers:node16.5.0-chrome94-ff93
-    name: E2E on Tests
+    container: cypress/browsers:node18.12.0-chrome103-ff107
+    name: E2E Tests
     strategy:
       matrix:
         browser: ['chrome', 'firefox']
     steps:
-      - uses: actions/checkout@v2
-      - uses: pnpm/action-setup@v2.2.2
-        with:
-          version: 7.4.0
-          run_install: true
-      - name: Cypress run
-        uses: cypress-io/github-action@v4
-        with:
-          install: false
-          browser: ${{ matrix.browser }}
-          env: NEXT_PUBLIC_USE_LOCAL_CLIENT=1
-          build: pnpm build:e2e
-          command: pnpm test:e2e
-          start: pnpm start:e2e
-          wait-on: 'http://localhost:3000'
-          # wait for 3 minutes for the server to respond
-          wait-on-timeout: 180
-      - uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: cypress-screenshots
-          path: cypress/screenshots
-      # Test run video was always captured, so this action uses "always()" condition
-      - uses: actions/upload-artifact@v2
-        if: always()
-        with:
-          name: cypress-videos
-          path: cypress/videos
+      - name: Skip Cypress Tests
+        run: |
+          echo "Skipping Cypress tests as we transition to a new playwright"
```

Comment:
**suggestion**: Should a link be included to #4580?
```suggestion
          echo "Skipping Cypress tests as we transition to Playwright as per https://github.com/tinacms/tinacms/issues/4580"
```

Reply:
✅ ty